php-fpm/Dockerfile

```diff
@@ -47,7 +55,7 @@ RUN apt-get update -yqq && \
 # Optional Software's will only be installed if you set them to `true`
 # in the `docker-compose.yml` before the build.
 # Example:
-#   - INSTALL_ZIP_ARCHIVE=true
```

Comment:
changed the comment to some other env var because if we will leave it like this it will mislead people